course/templates/course/create_review.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'base_home.html' %}
```

Comment:
Maybe extend the loggedInPage so the side navbar will appear

Reply:
Done